src/Core/Protocol.h

```diff
@@ -104,32 +106,9 @@ namespace Protocol
         /// would always be true because of compiler optimization. That would lead to out-of-bounds error
         /// if the packet is invalid.
         /// See https://www.securecoding.cert.org/confluence/display/cplusplus/INT36-CPP.+Do+not+use+out-of-range+enumeration+values
-        inline const char * toString(UInt64 packet)
+        inline std::string_view toString(UInt64 packet)
```

Comment:
AFAICT this is only used in error messages. Let's move it to the .cpp instead please, and avoid including magic_enum in a header

Reply:
Indeed! Actually I've cleaned it up a little bit more.